### ISSUE TITLE
Models？

### ISSUE THREAD

**Issue by jsjs0827** [yueruchen/Pedestrian-Synthesis-GAN#5]
Sorry to bother you, could you release some models in your datasets, and how to make the train data?

**Comment by yueruchen** [comment]
It’s easy to get the training data based on the samantic label in cityscapes dataset. Please download cityscapes dataset in the official website as we don’t have the copyright. And it is also easy to train it as we have released the code and hyper-parameters.
